From 64e6b10f98fe7226a2f41807268dae3afa80236d Mon Sep 17 00:00:00 2001
From: György Balló <ballogy@freestart.hu>
Date: Fri, 30 Sep 2011 21:48:40 +0000
Subject: data: add back check for gnome-shell before running gdm-shell session

Add back commit 4892c9447324f2248ce944f150abe3143f78eaa3

This commit does a quick check to make sure gnome-shell is
installed, and supports --gdm-mode. If either of those fail,
then fall back to the old greeter.

https://bugzilla.gnome.org/show_bug.cgi?id=660409
---
diff --git a/data/gdm-shell.session.in b/data/gdm-shell.session.in
index 3d408c3..496cc10 100644
--- a/data/gdm-shell.session.in
+++ b/data/gdm-shell.session.in
@@ -1,5 +1,5 @@
 [GNOME Session]
 Name=Display Manager
 RequiredComponents=gnome-shell;gnome-settings-daemon;
-IsRunnableHelper=@libexecdir@/gnome-session-check-accelerated
+IsRunnableHelper=bash -c 'gnome-shell --help | grep -q gdm-mode && @libexecdir@/gnome-session-check-accelerated'
 FallbackSession=gdm-fallback
--
cgit v0.9.0.2
